hack/bin/run-local.sh

```diff
@@ -115,7 +135,7 @@ function main {
   pauseInClusterOperator
 
   echo "### Running operator"
-  go run . "${OLLYOPARGS[@]}" -grafana-url http://localhost:3000 -monitoring-metrics-query-url http://localhost:8180/prometheus -alertmanager-url http://localhost:8181
+  go run ./... "${OLLYOPARGS[@]}" --webhook-cert-path="$CERTSDIR" -grafana-url http://localhost:3000 -monitoring-metrics-query-url http://localhost:8180/prometheus -alertmanager-url http://localhost:8181
```

Comment:
We don't want to just run "anything", we want the main.go sitting in `./cmd`.

```suggestion
  go run ./cmd "${OLLYOPARGS[@]}" --webhook-cert-path="$CERTSDIR" -grafana-url http://localhost:3000 -monitoring-metrics-query-url http://localhost:8180/prometheus -alertmanager-url http://localhost:8181
```